setup/requirements.md

```diff
@@ -49,12 +49,12 @@ The use of Coder deployments requires a license that's emailed to you.
 
 ### Restrictions
 
-Deployments using the free level of Coder (1-10 users):
+Deployments using the free 60-day free trial of Coder:
```

Comment:
**question:** Is it necessary to quantify here? I imagine if we drop the quantifier, it's less prone to becoming outdated without sacrificing clarity. Not sure though, thought I'd just ask

Reply:
so `deployments using the free trial of Coder`?

Reply:
Good point. Your phrasing works better. Main thing is to get rid of the 1-10 users language since that is no longer the case.